src/views/CreateType.vue

```diff
@@ -208,17 +227,46 @@ export default class CreateType extends Mixins(Translate) {
   }
 
   public editAttribute(name: string): void {
+    this.setEponymousAttribute();
     this.currentlyEditingAttribute = this.newResourceType.attributes.findIndex(
       (attribute) => attribute.name === name,
     );
   }
 
   public addAttribute(): void {
+    this.setEponymousAttribute();
     this.newResourceType.attributes.push(CreateType.emptyResourceTypeAttribute());
     this.currentlyEditingAttribute = this.newResourceType.attributes.length - 1;
   }
 
+  public deleteAttribute(): void {
+    if (this.editingAttribute.isEponymousAttribute) {
+      this.newResourceType.eponymousAttribute = undefined;
+    }
+    this.newResourceType.attributes.splice(this.currentlyEditingAttribute, 1);
+    this.currentlyEditingAttribute = -1;
+  }
+
+  public setEponymousAttribute(): void {
+    if (!this.editingAttribute) {
+      return;
+    }
+
+    if (!this.editingAttribute.isEponymousAttribute) {
+      return;
+    }
+
+    this.newResourceType.attributes.forEach((attribute) => {
+      if (attribute.name === this.editingAttribute.name) {
+        return;
+      }
+      attribute.isEponymousAttribute = false;
+    });
+    this.newResourceType.eponymousAttribute = this.editingAttribute.name;
+  }
+
   public saveAttribute(): void {
+    this.setEponymousAttribute();
```

Comment:
when saving a attribute after disabling the Eponymous-toggle it is still classified as "naming"